sqlobject/src/main/java/org/jdbi/v3/sqlobject/customizers/BindIn.java

```diff
@@ -36,14 +36,20 @@
 import org.jdbi.v3.sqlobject.SqlStatementCustomizingAnnotation;
 
 /**
- * Binds an Iterable or array/varargs of any type to a placeholder as a comma-separated list (e.g. for WHERE X IN (...) query statements).
+ * Binds each value in the annotated {@link Iterable} or array/varargs argument, and defines a named attribute as a
+ * comma-separated list of each bound parameter name. Common use cases:
  *
- * Throws IllegalArgumentException if the argument is not an array or Iterable. How null and empty collections are handled can be configured with onEmpty:EmptyHandling - throws IllegalArgumentException by default.
+ * <pre>
+ * &#64;SqlQuery("SELECT * FROM THINGS WHERE ID IN (&lt;ids&gt;)")
```

Comment:
Could you convert `things` in the lower case? I think this would be easier for the reader to distinguish between SQL statements and database objects.